### PR TITLE
Docs for optimizing mass data loading

### PR DESCRIPTION
**Pull request by allwefantasy**
Add docs for Optimizing Mass Data Loading in docs/useful-tips-on-carbondata.md

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1653/

**Comment by chenliang613** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1671/

File: docs/useful-tips-on-carbondata.md [hunk 1]
```diff
@@ -161,6 +162,34 @@ excessive memory usage.
 1 million.
 
 
+## Optimizing Mass Data Loading
```
Comment (chenliang613) [inline docs/useful-tips-on-carbondata.md]: How about  using this description: Configuration for Optimizing Data Loading performance for Massive Data

**Comment by chenliang613** [comment]
@sgururajshetty  please review it.

File: docs/useful-tips-on-carbondata.md (hunk 1 above)
Comment (allwefantasy) [inline docs/useful-tips-on-carbondata.md]: OK

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1675/

**Comment by allwefantasy** [comment]
@sgururajshetty I have modified the paragraph.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1692/

**Comment by chenliang613** [comment]
@allwefantasy  can you squash all commits into one commit.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1779/

**Comment by allwefantasy** [comment]
check new PR :   https://github.com/apache/incubator-carbondata/pull/845

**Pull request closed without merging**